ta/Makefile

```diff
@@ -8,6 +8,8 @@ ifeq ($(out-dir),)
 $(error invalid output directory (O=$(O)))
 endif
 
+-include $(TA_DEV_KIT_DIR)/host_include/conf.mk
```

Comment:
Well, it makes *some* sense, but not that much actually ;)
If `conf.mk` is missing, one can still pass the config options on the command line for instance or export them from a higher-level Makefile (I suppose that's why @etienne-lms could build the SDP app without this patch!). But I see no circumstance where `host_include` or `conf.mk` would not be available in the devkit when one is building `optee_test`, so yeah, better remove the dash so that any issue is detected early.